test/features/deploy_travel_agency.feature

```diff
@@ -84,75 +82,73 @@ Feature: Deploy Travel agency service and verify its functionality
+          "email" : "jan.kowalski@example.com",
+          "nationality" : "Polish",
+          "address" : {
+            "street" : "polna",
+            "city" : "Krakow",
+            "zipCode" : "32000",
+            "country" : "Poland"
+          }
+        },
+        "trip" : {
+          "city" : "New York",
+          "country" : "US",
+          "begin" : "2019-12-10T00:00:00.000+02:00",
+          "end" : "2019-12-15T00:00:00.000+02:00"
+        }
+      }
       """
     And Service "travels" contains 1 instance of process with name "travels"
     And Service "travels" contains 1 task of process with name "travels" and task name "VisaApplication"
     And Complete "VisaApplication" task on service "travels" and process with name "travels" with body:
-	  """json
-	{
-		"visaApplication" : {
-			"firstName" : "Jan",
-			"lastName" : "Kowalski",
-			"nationality" : "Polish",
-			"city" : "New York",
-			"country" : "US",
-			"passportNumber" : "ABC09876",
-			"duration" : 25
-		}
-	}
-	  """
+      """json
+      {
+        "visaApplication" : {
+          "firstName" : "Jan",
+          "lastName" : "Kowalski",
+          "nationality" : "Polish",
+          "city" : "New York",
+          "country" : "US",
+          "passportNumber" : "ABC09876",
+          "duration" : 25
+        }
+      }
+      """
 	And Service "visas" contains 1 instance of process with name "visaApplications" within 1 minutes
 	And Service "visas" contains 1 task of process with name "visaApplications" and task name "ApplicationApproval"
 	And Complete "ApplicationApproval" task on service "visas" and process with name "visaApplications" with body:
 	  """json
-	{
-		"application" : {
-			"firstName" : "Jan",
-			"lastName" : "Kowalski",
-			"nationality" : "Polish",
-			"city" : "New York",
-			"country" : "US",
-			"passportNumber" : "ABC09876",
-			"duration" : 25,
-			"approved" : true
-		}
-	}
-	  """
-    And Service "travels" contains 1 task of process with name "travels" and task name "ConfirmTravel"
-    And Complete "ConfirmTravel" task on service "travels" and process with name "travels" with body:
-	  """json
-	  {}
+    {
+      "application" : {
+        "firstName" : "Jan",
+        "lastName" : "Kowalski",
+        "nationality" : "Polish",
+        "city" : "New York",
+        "country" : "US",
+        "passportNumber" : "ABC09876",
+        "duration" : 25,
+        "approved" : true
+      }
+    }
 	  """
+  And Service "travels" contains 1 task of process with name "travels" and task name "ConfirmTravel"
+  And Complete "ConfirmTravel" task on service "travels" and process with name "travels" with body:
+    """json
+    {}
+    """
 
-    Then Service "travels" with process name "travels" is available
+  Then Service "travels" with process name "travels" is available
 
   Examples:
-      | native   | minutes | build-request-cpu | 
-      | disabled | 10      | 1                 | 
+    | native   | minutes | build-request-cpu | 
```

Comment:
There are some indentation problems in the lines before (starting line 126), can you have a quick look ?

Reply:
Not sure why I see indentations problems locally, but then here is right the opposite (I confirmed I use spaces configuration...). However, I reverted the format changes.